examples/openshift/openshift-values.yaml

```diff
@@ -1,7 +1,5 @@
 global:
   openshift: true
-  spire:
-    jwtIssuer: https://oidc-discovery.$APP_SUBDOMAIN
```

Comment:
Move things the user must edit to the examples-your-values.yaml, same with ingress hosts rules.